File: Exercises/rb101-rb109_small_problems/easy_8/madlibs.rb
# frozen_string_literal: true

def prompt(message)
  puts "=> #{message}"
end

prompt 'Enter a noun:'
noun = gets.chomp

prompt 'Enter a verb:'
verb = gets.chomp

prompt 'Enter an adjective:'
adjective = gets.chomp

prompt 'Enter an adverb:'
adverb = gets.chomp

prompt "Today you will #{verb} a #{adjective} #{noun} #{adverb}."

File: js210/exercises/easy_2/madlibs.js
const rlSync = require('readline-sync');
const noun = rlSync.question('Enter a noun: ');
const verb = rlSync.question('Enter a verb: ');
const adjective = rlSync.question('Enter an adjective: ');
const adverb = rlSync.question('Enter an adverb: ')
const libs = [
  `Do you ${verb} your ${adjective} ${noun} ${adverb}? That's hilarious!`,
  `Never ${verb} ${adverb} with a ${adjective} ${noun}.`,
  `Sometimes you just want to ${adverb} ${verb} while watching a ${adjective} ${noun} sing.`
];

let libIndex = Math.floor(Math.random() * libs.length);
console.log(libs[libIndex]);
